src/test/java/com/exasol/sql/ddl/drop/TestDropSchema.java

```diff
@@ -0,0 +1,45 @@
+package com.exasol.sql.ddl.drop;
+
+import com.exasol.sql.StatementFactory;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.*;
+
+class TestDropSchema {
+    private static final String TEST_SCHEMA_NAME = "test schema name";
+    private DropSchema dropSchema;
+
+    @BeforeEach
+    void setUp() {
+        dropSchema = StatementFactory.getInstance().dropSchema(TEST_SCHEMA_NAME);
+    }
+
+    @Test
+    void getTableName() {
+        assertThat(this.dropSchema.getSchemaName(), equalTo(TEST_SCHEMA_NAME));
+    }
+
+    @Test
+    void ifExists() {
```

Comment:
Split into two tests please. One test the default setting, the other one the setter. This way you have only one assert in each.

Reply:
fixed